fetch.bs

```diff
@@ -1519,6 +1519,14 @@ user-agent-defined object). Unless otherwise stated it is null.
 <a lt=fetch for=/>fetching</a>. It provides a convenient way for standards to not have to set
 <a for=/>request</a>'s <a for=request>origin</a>.
 
+<p>A <a for=/>request</a> has an associated <dfn export for=request
+id=concept-request-policy-container>policy container</dfn>, which is "<code>client</code>" or a <a
+for=/>policy container</a>. Unless stated otherwise it is "<code>client</code>".
+
+<p class="note no-backref">"<code>client</code>" is changed to a <a for=/>policy container</a>
```

Comment:
This can be `class=note`. I should do a cleanup PR to remove all the no-backref stuff.